be2-scala/src/main/scala/ch/epfl/pop/model/objects/Transaction.scala

```diff
@@ -30,7 +30,32 @@ final case class Transaction(
   Version: Int,
   TxIn: List[TxIn],
   TxOut: List[TxOut],
-  )
+  LockTime: Int,
+) {
+  lazy val transaction_id = {
+    import scala.util.chaining._
```

Comment:
I would probably still leave the imports at the top of the file for clarity unless there is a very specific reason for this